src/coinbase/address/developer_address.ts

```diff
@@ -0,0 +1,364 @@
+import { Decimal } from "decimal.js";
+import { ethers } from "ethers";
+import { Address } from "../address";
+import { Address as AddressModel } from "../../client";
+import { Asset } from "../asset";
+import { Balance } from "../balance";
+import { BalanceMap } from "../balance_map";
+import { Coinbase } from "../coinbase";
+import { ArgumentError, InternalError } from "../errors";
+import { FaucetTransaction } from "../faucet_transaction";
+import { Trade } from "../trade";
+import { Transfer } from "../transfer";
+import { Amount, Destination, TransferStatus } from "../types";
+import { delay } from "../utils";
+import { Wallet as WalletClass } from "../wallet";
+
+/**
+ * A representation of a blockchain address, which is a developer-controlled account on a network.
+ */
+export class DeveloperAddress extends Address {
```

Comment:
This one should be `WalletAddress`. Ruby reference: https://github.com/coinbase/coinbase-sdk-ruby/pull/86/files#diff-1d829116f97d14de35db53f5a0a130c39e90e5785185b15f9f6efceea7ecf7ad

Reply:
Yeah good call, renamed it across of areas.